### PR TITLE
Add tests page

### PR DESCRIPTION
**Pull request by pio2398**
Task 11

**Description**

Add table with exams lists. Probably I shouldn't use axis this way but I don't have idea how add token globally to requests.

**Content**

- [x] Component
- [] Component tests
- [ ] Refactor the code

**Is the feature tested?**

- [X] Yes (limited)
- [] Not yet
- [ ] Nothing to test

**Additional changes**

Describe some additional changes that you've added with this pull request

**To discuss**

Some things you want to discuss with other contributors during the code review

### CODE REVIEW

**Review by PawelKapusta — APPROVED** [review]

**Review by bkstud — APPROVED** [review]

File: src/screens/Tests.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  }},
+];
+
+const Tests = () => {
+    const [exams, setExams] = useState([])
+  
+
+    useEffect(() => {
+        const token = JSON.parse(localStorage.getItem("token"));
+        const http = axios.create({
```
Comment (bkstud) [inline src/screens/Tests.jsx]: Please do not recreate axios object  instead use the one generic already created at src/lib/axios.js

Comment (pio2398) [reply src/screens/Tests.jsx]: but how pass token to global object?

Comment (bkstud) [reply src/screens/Tests.jsx]: I suggest checking if  token added as parameter to `get("/exams")` works or leaving it this way, but just change the url "localhost" to one taken from src/lib/axios.js as global variable URL.

Comment (pio2398) [reply src/screens/Tests.jsx]: Should be OK now, thanks.

**Pull request merged**